integrations-and-supported-tools/kedro/scripts/kedro_neptune_advanced/src/kedro_neptune_advanced/pipelines/data_science/nodes.py

```diff
@@ -0,0 +1,116 @@
+# BE LIABLE FOR ANY CLAIM, DAMAGES, OR OTHER LIABILITY, WHETHER IN AN
+# ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF, OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+# The QuantumBlack Visual Analytics Limited ("QuantumBlack") name and logo
+# (either separately or in combination, "QuantumBlack Trademarks") are
+# trademarks of QuantumBlack. The License does not grant you any right or
+# license to the QuantumBlack Trademarks. You may not use the QuantumBlack
+# Trademarks or any confusingly similar mark as a trademark for your product,
+# or use the QuantumBlack Trademarks in any other manner that might cause
+# confusion in the marketplace, including but not limited to in advertising,
+# on websites, or on software.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Example code for the nodes in the example pipeline. This code is meant
+just for illustrating basic Kedro features.
+
+Delete this when you start working on your own Kedro project.
+"""
+# pylint: disable=invalid-name
+
+import logging
+import matplotlib.pyplot as plt
+import neptune.new as neptune
+import numpy as np
+import pandas as pd
+from scikitplot.metrics import plot_roc_curve, plot_precision_recall_curve
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.metrics import accuracy_score
+from sklearn.neural_network import MLPClassifier
+from typing import Any, Dict
+
+
+def train_rf_model(
+        train_x: pd.DataFrame, train_y: pd.DataFrame, parameters: Dict[str, Any]
+):
+    """Node for training Random Forest model"""
+    max_depth = parameters["rf_max_depth"]
+    n_estimators = parameters["rf_n_estimators"]
+    max_features = parameters["rf_max_features"]
+
+    clf = RandomForestClassifier(max_depth=max_depth,
+                                 n_estimators=n_estimators,
+                                 max_features=max_features)
+    clf.fit(train_x, train_y.idxmax(axis=1))
+
+    return clf
+
+
+def train_mlp_model(
+        train_x: pd.DataFrame, train_y: pd.DataFrame, parameters: Dict[str, Any]
+):
+    """Node for training MLP model"""
+    alpha = parameters["mlp_alpha"]
+    max_iter = parameters["mlp_max_iter"]
+
+    clf = MLPClassifier(alpha=alpha,
+                        max_iter=max_iter)
+    clf.fit(train_x, train_y)
+
+    return clf
+
+
+def get_predictions(rf_model: RandomForestClassifier, mlp_model: MLPClassifier,
+                    test_x: pd.DataFrame) -> Dict[str, Any]:
+    """Node for making predictions given a pre-trained model and a test set."""
+    predictions = {}
+    for name, model in zip(['rf', 'mlp'], [rf_model, mlp_model]):
+        y_pred = model.predict_proba(test_x).tolist()
+        predictions[name] = y_pred
+
+    return predictions
+
+
+def evaluate_models(predictions: dict, test_y: pd.DataFrame,
+                    neptune_run: neptune.run.Handler):
```

Comment:
Not sure, but it probably should be 
`neptune_run: neptune.run.Run` -> it's neptune Run object that you use for logging.